gulpfile.js

```diff
@@ -0,0 +1,171 @@
+};
+
+gulp.task('default', function (cb) {
+    runSequence('clean', 'bower', 'libs', 'build', cb);
+});
+
+gulp.task('clean', function () {
+    return gulp.src(buildDir + '/*', { read: false })
+        .pipe(clean({ force: true }));
+});
+
+gulp.task('bower', function () {
+    return bower()
+        .pipe(gulp.dest(libDir))
+});
+
+gulp.task('libs', function () {
+    return gulp.src(libDir + '/**/*.min.js')
+        .pipe(gulp.dest(clientLibDir));
+});
+
+gulp.task('build', function (cb) {
+    runSequence('copy-static', 'templates', 'scripts', 'css', cb);
+});
+
+gulp.task('copy-static', function (cb) {
+    runSequence('copy-html', 'copy-images', cb);
+});
+
+gulp.task('copy-html', function () {
+    return gulp.src(path.html)
+        .pipe(gulpif(isProd, htmlmin({ collapseWhitespace: true })))
+        .pipe(gulp.dest(clientDir))
+        .pipe(livereload());
+});
+
+gulp.task('copy-images', function () {
+    return gulp.src(path.images)
+        .pipe(gulp.dest(clientDir))
+        .pipe(livereload());
+});
+
+gulp.task('css', function () {
+    return gulp.src(path.css)
+        .pipe(concat('styles.css'))
+        .pipe(less())
+        .pipe(sourcemaps.init())
+        .pipe(autoprefixer())
+        .pipe(gulpif(isProd, cssnano()))
+        .pipe(sourcemaps.write(cssMapDir))
+        .pipe(gulp.dest(clientStaticDir))
+        .pipe(livereload());
+});
+
+gulp.task('templates', function () {
+    return gulp.src(path.templates)
+        .pipe(handlebars())
+        .pipe(wrap('Handlebars.template(<%= contents %>)'))
+        .pipe(declare({
+            namespace: 'App.templates',
+            noRedeclare: true
+        }))
+        .pipe(concat('templates.js'))
+        .pipe(sourcemaps.init())
+        .pipe(gulpif(isProd, uglify()))
+        .pipe(sourcemaps.write(jsMapDir))
+        .pipe(gulp.dest(clientDir))
+        .pipe(livereload());
+});
+
+gulp.task('scripts', function () {
+    gulp.src(path.scripts)
+        .pipe(concat('app.js'))
+        .pipe(sourcemaps.init())
+        .pipe(gulpif(isProd, uglify()))
+        .pipe(sourcemaps.write(jsMapDir))
+        .pipe(gulp.dest(clientDir))
+        .pipe(livereload());
+});
+
+gulp.task('watch', function () {
+    livereload.listen();
+    gulp.watch(path.scripts, ['scripts']);
+    gulp.watch(path.templates, ['templates']);
+    gulp.watch(path.css, ['css']);
+    gulp.watch(path.html, ['copy-html']);
+    gulp.watch(path.images, ['copy-images']);
+});
+
+gulp.task('style', function (cb) {
+    runSequence('style-jscs', 'style-csscomb', 'style-jshint', 'style-htmlhint', cb);
+});
+
+function gitmodified() {
+    return gitmodified0(['added', 'modified', 'untracked', 'renamed', 'copied', 'updated but unmerged']);
+}
+
+gulp.task('style-jscs', function () {
+    return gulp.src(path.scripts)
+        .pipe(gulpif(!isAll, gitmodified()))
+        .pipe(debug({ title: 'jscs' }))
+        .pipe(jscs({ fix: true }))
+        .pipe(gulp.dest(srcDir));
+});
+
```

Comment:
checked by modifying configurations and checking gulp messages. everything works as expected